hypervisor/arch/x86/Kconfig

```diff
@@ -220,13 +220,25 @@ config RAM_START
 	  of relocating its symbols to where it is placed at, and thus the
 	  bootloader may not place the hypervisor at this specific address.
 
-config RAM_SIZE
-	hex "Size of the RAM region assigned to the hypervisor"
-	default 0x02000000
+config HV_RAM_SIZE
+	hex "Size of the RAM region used by the hypervisor"
+	default 0x04000000
 	help
-	  A 64-bit integer indicating the size of RAM assigned to the
-	  hypervisor. It is ensured at link time that the footprint of the
-	  hypervisor does not exceed this size.
+	  A 64-bit integer indicating the size of RAM used by the hypervisor.
+	  It is ensured at link time that the footprint of the hypervisor
+	  does not exceed this size.
+
+config PLATFORM_RAM_SIZE
+	hex "Size of the RAM"
+	default 0x200000000
+	help
+	  A 64-bit integer indicating the size of the RAM (not included the MMIO).
+
+config PLATFORM_LO_MMIO_SIZE
```

Comment:
No need to configure it so far. Make it fixed in code.  1GB or 2GB is fine
If we want, we should set the HI_MMIO_SIZE